lexis360/parser.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Lexis 360
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+  let rawUrl = ec.url;
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+  //console.error('rawUrl: ' + rawUrl);
+
+  let match;
+
+  //if ((match = /^\/platform\/path\/to\/(document\-([0-9]+)\-test\.pdf)$/i.exec(path)) !== null) {
+  if ((match = /^\/Document\/([\w]+)\/([\w\-]+)$/i.exec(path)) !== null) {
+    //http://www.lexis360.fr/Document/droit_fiscal_29_septembre_2016_n_39/EQoofIo2izxXAW-NWTzsDkSV01q3G5HXt9qLfVgetxD6vmamzHEa7qFKMR2TVPp30?data=c0luZGV4PTEmckNvdW50PTE3NjIm&rndNum=869697884&tsid=search7_
+    result.rtype    = 'TOC';
+    result.mime     = 'HTML';
+    result.unitid   = param.rndNum;
```

Comment:
In this case, the URL is correctly parsed, allowing us working with the parsedUrl